packages/yourloops/lib/auth/user-api.ts

```diff
@@ -0,0 +1,70 @@
+import { Preferences, Profile, Settings } from "../../models/shoreline";
+import HttpService from "../../services/http";
+import { HttpHeaderKeys } from "../../models/api";
+
+export default class UserApi {
+
+  static async getShorelineAccessToken(email: string): Promise<string> {
+    try {
+      const { headers } = await HttpService.post({ url: `auth/hack/user/${email}` });
+      return headers[HttpHeaderKeys.sessionToken];
+    } catch (err) {
+      console.log("This profile doesn't exists");
+      return "no-token";
+    }
+  }
+
+  static async getProfile(userId: string): Promise<Profile | undefined> {
+    try {
+      const { data } = await HttpService.get<Profile>({ url: `/metadata/${userId}/profile` });
+      return data;
+    } catch (err) {
+      console.log(`No settings for ${userId}`);
+      return undefined;
```

Comment:
Shouldn't we throw an error instead? (I imagine that it was handled like this before though)